src/models/economic/AALLocalitySummary.js

```diff
@@ -0,0 +1,20 @@
+const { isArray } = require('lodash');
```

Comment:
can we pull `isArray` vs calling in the lib directly?

``` const isArray = require('lodash/isArray');```

Reply:
that's what code formatter prefers, I guess it's a standard?